tests/test_flow/fakes.py

```diff
@@ -205,28 +201,27 @@ class InstrumentedFilesystem:
     ("get") operations.
     """
 
-    def __init__(self, wrapped_fs, make_list):
+    def __init__(
+        self,
+        wrapped_fs,
+        shared_downloaded_list,
```

Comment:
Huh. I'm confused by how this would happen, but maybe I need to think more about this. When is the serialization of the `make_list` function happening?

Reply:
Hmm, I do not remember what issues I encountered earlier, but switching back to passing `make_list` works now.